docs/WebServices-Methods.md

```diff
@@ -1460,6 +1461,31 @@ REST endpoint: `POST /rest/v3/ecies/decryptor`
 | `String` | `secretKey` | Base64 encoded secret key for ECIES |
 | `String` | `sharedInfo2` | The sharedInfo2 parameter for ECIES |
 
+### Method 'getEciesEncryptor'
+
+Get ECIES encryptor data for request/response encryption on intermediate server.
+
+#### Request
+
+REST endpoint: `POST /rest/v3/ecies/encryptor`
+
+`GetEciesEncryptorRequest`
+
+| Type | Name | Description |
+|------|------|-------------|
+| `String` | `activationId` | A UUID4 identifier of an activation (used only in activation scope, use null value in application scope) |
+| `String` | `applicationKey` | A key (identifier) of an application, associated with given application version |
+| `String` | `ephemeralPublicKey` | A base64 encoded ephemeral public key for ECIES |
```

Comment:
Do we have use-case for this? In `powerauth-restful-integration` project we don't use this API at all so I wonder why we need such API.

If it provide the same data as `/ecies/decryptor` then let's remove it as redundant.

Reply:
You are right, we can use the original endpoint. I have not realized the returned data is the same.